application/classes/Ushahidi/Formatter/Post/CSV.php

```diff
@@ -43,23 +54,17 @@ protected function generateCSVRecords($records)
 		 */
 		$headingColumns = $this->getCSVHeading($records);
```

Comment:
Are we still thinking of making CSV heading an optional feature, if we're going to do partial CSV exports and later concat them together? Apologies if this exists somewhere else and I don't see it.

Reply:
Yes, I can add a means to do that, will do

Reply:
done